code/js/modules/Sitelist.js

```diff
@@ -126,6 +126,7 @@
       "shortorange": { name: "ShortOrange", url: "http://www.shortorange.com" },
       "shuffler": { name: "Shuffler.fm", url: "http://www.shuffler.fm" },
       "slacker": { name: "Slacker", url: "http://www.slacker.com" },
+      "somafm": { name: "SomaFM", url: "http://somafm.com/player/#/now-playing/" },
```

Comment:
Please change to just `http://somafm.com` (`url` is only used to link to the sites, so I like to keep it as a link to the "root" music site)